src/ansys/dpf/post/result_workflows/_sub_workflows.py

```diff
@@ -266,7 +266,7 @@ def _create_sweeping_phase_workflow(
             raise ValueError("Argument sweeping_phase must be a float.")
         sweeping_op = create_operator_callable(name="sweeping_phase_fc")
         sweeping_op.connect(2, sweeping_phase)
```

Comment:
Not sure why this only popped up now. Probably this workflow is not tested otherwise.